### DISCUSSION TITLE
Save the created .svg as an Image

### DISCUSSION THREAD

**Discussion by Tuskiurhil** [adrianschlatter/ppf.datamatrix#5] [Q&A]
Hello Adrian,

is there a way to save the created DataMatrix once it's been created with "DataMatrix(input)" ?

**Comment by adrianschlatter** [comment]
Simply call the .svg() method and you get the datamatrix as Scalable Vector-Graphics.

**Comment by yatesdr** [comment]
For anybody else with this question, in case it's not obvious you can just write it to an SVG file:

```
my_code = DataMatrix("abcdefg")
svg = my_code.svg()
with open('output.svg','w') as f:
     f.write(svg)
```
